### ISSUE TITLE
Update CODEOWNERS workflow is failing

### ISSUE THREAD

**Issue by MysteryBlokHed** [DefinitelyTyped/DefinitelyTyped#62528]
### Discussed in https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/62525

<div type='discussions-op-text'>

<sup>Originally posted by **MysteryBlokHed** October  2, 2022</sup>
The workflow to update the codeowners has been failing for a little while: <https://github.com/DefinitelyTyped/DefinitelyTyped/actions/workflows/UpdateCodeowners.yml>

Here are the logs from the [latest run](https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/3126099266) as of this discussion being created:

```text
> definitely-typed@0.0.3 update-codeowners
> node scripts/update-codeowners.js

git checkout .
Updated 0 paths from the index
Using local DefinitelyTyped at .
Parsing definitions...
Found 8677 packages.
Parsing in 2 child process(es)...
Parsing took 34.742 s
git add .github/CODEOWNERS
git pull
Already up to date.
git commit -m "🤖 Update CODEOWNERS"
[master da0c32be5a] 🤖 Update CODEOWNERS
 1 file changed, 8986 insertions(+), 8461 deletions(-)
git push
remote: Permission to DefinitelyTyped/DefinitelyTyped.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/DefinitelyTyped/DefinitelyTyped/': The requested URL returned error: 403
Error: git push failed: null
    at runSequence (file:///home/runner/work/DefinitelyTyped/DefinitelyTyped/scripts/update-codeowners.js:51:19)
    at file:///home/runner/work/DefinitelyTyped/DefinitelyTyped/scripts/update-codeowners.js:32:9
Error: Process completed with exit code 1.
```

The workflow doesn't seem to have permission to push to this repository. Maybe its token got revoked and needs to be updated?

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/081b2dc558780b15ecfd508890f0729e6598391d/.github/workflows/UpdateCodeowners.yml#L32-L34></div>

**Comment by MysteryBlokHed** [comment]
@sandersn Sorry to ping you directly, but I’ve seen you review infrastructure changes and want to make sure you’re aware of this

**Comment by MysteryBlokHed** [comment]
Fixed as of 92f4e5e9d021ff808e033e7c701f36e1c18e9997
